glass-easel/tests/types/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "types": []
```

Comment:
This is here to avoid global typings leaked from `@types/jest` and `@types/node`. The type tests are not executed, hence it could be very hard to catch mistakes coming from global typings. In this context, it is better to have less accidental typings around.